src/Workspaces/CoreTest/CodeCleanup/ReduceTokenTests.cs

```diff
@@ -1823,6 +1823,64 @@ End Sub
             }
         }
 
+        [Fact]
+        [Trait(Traits.Feature, Traits.Features.ReduceTokens)]
+        public void ReduceIntegerLiteralWithLeadingZeros()
+        {
+            var code = @"[|
+Module Program
+    Sub Main(args As String())
+        Const i0 As Integer = 0060
+        Const i1 As Integer = 0060%
+        Const i2 As Integer = &H006F
+        Const i3 As Integer = &O0060
+        Const i4 As Integer = 0060I
+        Const i5 As Integer = -0060
+        Const l0 As Long = 0060L
+        Const l1 As Long = 0060&
+        Const s0 As Short = 0060S
```

Comment:
Can we have some tests around the error cases? I can think of overflow/underflow, and also literals around min/max values to make sure we don't have problems there.